### PR TITLE
fix: update placeholder regex to match multiple occurences in same st…

### PR DESCRIPTION
**Pull request by llavaud**
### Description
Update placeholder regex to support multiple occurences in the same string

**Fixes:** #144 

### Checklist
Please make sure that your PR fulfills the following requirements:
- [x] Reviewed the guidelines for contributing to this repository
- [x] The commit message follows the [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0/#summary).
- [x] Tests for the changes have been updated
- [ ] Docs have been added / updated

### Type of Change

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] New tests
- [ ] Build/CI related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

### CODE REVIEW

**Comment by llavaud** [comment]
I will look if I can write a test tonight ! :)

File: pkg/kube/util.go [hunk 1]
```diff
@@ -15,8 +15,8 @@ import (
 )
 
 var genericPlaceholder, _ = regexp.Compile(`(?mU)<(.*)>`)
-var specificPathPlaceholder, _ = regexp.Compile(`(?mU)<path:(.+?)\#(.+?)>`)
-var indivPlaceholderSyntax, _ = regexp.Compile(`(?mU)path:(.+?)\#(.+?)`)
+var specificPathPlaceholder, _ = regexp.Compile(`(?mU)<path:(.+)\#(.+)>`)
+var indivPlaceholderSyntax, _ = regexp.Compile(`(?mU)path:(.+)\#(.+)`)
```
Comment (werne2j) [inline pkg/kube/util.go:19]: I dont think this line should change. I would revert it back and leave `specificPathPlaceholder` as the only changed line

**Review by werne2j — CHANGES_REQUESTED** [review]

File: pkg/kube/util.go (hunk 1 above)
Comment (llavaud) [reply pkg/kube/util.go:19]: it must not be a problem to leave it like it was but no need to be greedy here too. as you wish.

**Comment by jkayani** [comment]
@llavaud Thanks for the PR! Hopefully updating the existing tests/adding new ones isn't too hard, e.g, fixing this one:

```
=== RUN   TestMain/will_replace_templates_from_local_vault
2021-06-14T13:18:55.967Z [DEBUG] core0.identity: creating a new entity: alias="id:"3e0f62a7-2706-0976-f0e3-04318cfba481" canonical_id:"4f8b83fb-012c-6fac-c37e-0edff5c7ec9d" mount_type:"approle" mount_accessor:"auth_approle_2ac48815" mount_path:"auth/approle/" metadata:{key:"role_name" value:"role1"} name:"1e260ae6-c205-80ee-ad95-4452cd4f627c" creation_time:{seconds:1623676735 nanos:967905631} last_update_time:{seconds:1623676735 nanos:967905631} namespace_id:"root""
Error: Replace: could not replace all placeholders in Template:
replaceString: missing Vault value for placeholder path:secret/testing#secret-var-value in string SECRET_VAR: <path:secret/testing#secret-var-value>
```

**Review by werne2j — APPROVED** [review]
Looks good! Thanks for adding test cases!

**Review by jkayani — APPROVED** [review]
Awesome, thanks a lot @llavaud

**Comment by llavaud** [comment]
you plan to release a 1.1.1 quickly ?

**Comment by werne2j** [comment]
@llavaud yes, i will try to get 1.1.1 released today

**Pull request merged**